src/app/common/jeuxApi.ts

```diff
@@ -1,13 +1,29 @@
 export async function getData(url: string) {
-    const res = await fetch(url);
-    // The return value is *not* serialized
-    // You can return Date, Map, Set, etc.
-   
-    // Recommendation: handle errors
-    if (!res.ok) {
-      // This will activate the closest `error.js` Error Boundary
-      throw new Error('Failed to fetch data');
-    }
-   
+  const res = await fetch(url);
+  // The return value is *not* serialized
+  // You can return Date, Map, Set, etc.
+
+  // Recommendation: handle errors
+  if (!res.ok) {
+    // This will activate the closest `error.js` Error Boundary
+    throw new Error("Failed to fetch data");
```

Comment:
what's the difference in the body of the async function getData, beside the " " instead of ' ' in the error message ?